tests/tests.go

```diff
@@ -188,49 +187,34 @@ func setRange(rangeStart, rangeEnd string) error {
 	return nil
 }
 
-func DeleteLeaderManager() {
+func DeleteOneManager() {
```

Comment:
Is it "enough" to remove only one pod to cover our test cases?

Reply:
The original test deleted the leader to check for how much time it will take to regain kmp operation. In the sharding approach we have 2 running mgr so I think the test should be changed to check the the other kmp-mgr is ready in 0 time. This is what I'm checking here.

But I agree that it may be also interesting to add another test where we kill both pods to make sure there're ready in reasonable time. I will add it to the the list of tests added in the test task.